config.json

```diff
@@ -107,9 +107,15 @@
       "topics": []
     },
     {
-      "difficulty": 1,
+      "difficulty": 2,
```

Comment:
so, just so I'm clear, are you & @ferhatelmas OK if I continue adding difficulty & topics at the same time to exercises? I'll mostly go in order, unless I complete another exercise as I go, though it shouldn't take too long to catch up with myself. I'm just not familiar enough with the exercises to tackle all the difficulties at once. I'll also cross-reference with other tracks as I go.